paas/client.go

```diff
@@ -50,6 +52,38 @@ type CarrierClient struct {
 	Log           logr.Logger
 }
 
+func NewCarrierClient(flags *pflag.FlagSet) (*CarrierClient, func(), error) {
```

Comment:
This is exactly the code `wire` generated. For now I just moved it to a place where it is usually expected (after the struct definition). We can simplify the method later.